src/vs/base/browser/ui/actionbar/actionbar.css

```diff
@@ -5,7 +5,7 @@
 
 .monaco-action-bar {
 	text-align: right;
-	overflow: hidden;
+	/* overflow: hidden; */
```

Comment:
@sbatten really for all? Or only for the menu?

Reply:
@bpasero i'll create a new rule that's more specific, but I haven't encountered any issues yet with this change

Reply:
@sbatten if we can keep the change minimal to menus in general I think that would always be good unless we have a clear understanding what this rule was added in the first place